File: ympa_fsharp_giraffe_server/src/model/CampaignSettingsDTO.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic
open ympa_fsharp_giraffe_server.Model.CampaignSettingsLocalRegionDTO

module CampaignSettingsDTO =

  //#region CampaignSettingsDTO


  type CampaignSettingsDTO = {
    CountryRegion : int64;
    ShopName : string;
    ShowInContext : bool;
    ShowInPremium : bool;
    UseOpenStat : bool;
    LocalRegion : CampaignSettingsLocalRegionDTO;
  }
  //#endregion

File: src/main/kotlin/ympa_kotlin_wiremock_server/models/CampaignSettingsDTO.kt
@file:Suppress(
    "RemoveRedundantQualifierName",
    "unused",
)

package ympa_kotlin_wiremock_server.models

import com.fasterxml.jackson.annotation.JsonProperty

data class CampaignSettingsDTO(
    @field:JsonProperty("countryRegion")
    val countryRegion: kotlin.Long? = null,

    @field:JsonProperty("shopName")
    val shopName: kotlin.String? = null,

    @field:JsonProperty("showInContext")
    val showInContext: kotlin.Boolean? = null,

    @field:JsonProperty("showInPremium")
    val showInPremium: kotlin.Boolean? = null,

    @field:JsonProperty("useOpenStat")
    val useOpenStat: kotlin.Boolean? = null,

    @field:JsonProperty("localRegion")
    val localRegion: CampaignSettingsLocalRegionDTO? = null,

)
